inference-engine/samples/speech_sample/main.cpp

```diff
@@ -633,7 +505,34 @@ int main(int argc, char* argv[]) {
             return 0;
         }
 
-        std::vector<std::string> inputArkFiles;
+        BaseFile* file;
+        BaseFile* fileReferenceScores;
+        ArkFile* fileOutput = new ArkFile();
```

Comment:
type of output file should be set up via a new option. Let discuss it.

Reply:
We need to update README as well.

Reply:
Readme file was updated, you can look at it.